### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by jeisonjei** [eabrega/scroll-progress#9] [General]
Thank you for your job! I had problems with Bootstrap Scrollspy on my website, and this plugin after some code changes works perfectly!
I'll describe changes I made for myself - on my website I have collapsible side-menu and I use special scrollable container for the content, so body is not scrollable. Window.pageYOffset was always 0, alhough I'm not sure why. So, I replaced 
`window.pageYOffset`
to
`[myElement].scrollTop`
and it works very well even when hiding-showing side menu.
Thank you!

**Comment by eabrega** [comment]
Thank you for this comment!
My english not so good. I need some time for correct answer. ))
